build.ps1

```diff
@@ -225,5 +225,6 @@ $cakeArguments += $ScriptArgs
 
 # Start Cake
 Write-Host "Running build script..."
+&$CAKE_EXE --bootstrap
```

Comment:
What's bootstrap do?

Reply:
In the [Installation Section](https://www.gep13.co.uk/blog/introducing-cake.dotnettool.module#installation) it refers to needing to bootstrap cake for module inclusion.  It is a limitation that the Cake team is working to resolve so all you have to do is execute cake once in the future.